Resources/Prototypes/_Nuclear14/Entities/Objects/Consumable/Flora/wastelandflora.yml

```diff
@@ -312,24 +312,30 @@
 
 - type: entity
   parent: N14WastelandFlora
-  id: N14WastelandFloraWildJalapenoPepper
-  name: wild jalapeno pepper plant
-  description: A jalapeno pepper plant growing wild. It's healthy.
+  id: N14WastelandFloraWildGlowpepper #N14 change
```

Comment:
Put an entry for this in Migrations/nuclear14 so that it’s replaced on in progress maps.

Reply:
migrations entries added for both!